CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 ### New APIs
 * The Piwik JavaScript tracker has a new method `trackRequest` that allows you to send any tracking parameters to Piwik. For example  `_paq.push(['trackRequest', 'te=foo&bar=baz'])`
 
+### Breaking Changes
```

Comment:
I think should be "Internal change" instead as it's only breaking for developers running tests